externe_connecties/iv3_definitie_connectie.py

```diff
@@ -0,0 +1,72 @@
+from applicatie.logic.inlezen import ophalen_bestand_van_web
+from config.configurations import IV3_REPO_PATH
+import logging
+
+
+_logger = logging.getLogger(__file__)
+
+
+def geef_gebruikersvriendelijke_foutmeldingen(fouten):
+    """
+    Haal gebruikers vriendelijke vertalingen uit fouten.json voor fouten.
+    """
+    vriendelijke_fouten = []
+    vertalingen, fouten_bij_ophalen = ophalen_bestand_van_web(IV3_REPO_PATH, 'fouten.json', 'foutenbestand')
```

Comment:
Waarom heet het vertalingen?

Reply:
zal het hernoemen, het komt omdat er te veel dingen met fout in de naam zijn hier... eigenlijk wilde ik alle fouten_bij_ophalen (de tweede return argumenten) e.d. wegrefactoren en een mooie exception handling maken, maar dat is nogal wat werk. Denk je dat dat de moeite is? Daarmee voorkomen we alle geneste if-statements in routes.py.